src/pkg/build/conveyorPacker_debootstrap_test.go

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2018, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package build
+
+import (
+	"os"
+	"testing"
+)
+
+func TestDebootstrapConveyor(t *testing.T) {
+
+	//test must be run as root
+	if os.Getuid() != 0 {
```

Comment:
Just a `return` here? Isn't not being root a fatal error for Debootstrap? Or just not on the "test" part?

Reply:
As far as i know our tests with travis dont run with sudo so I did this to pass CI. I think it would be better to test this at a higher level where we call "sudo singularity build..." in a script